lib/floki/text_extractor.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Floki.TextExtractor do
+  @moduledoc false
+
+  @allowed_input_types [
+    nil,
```

Comment:
Why `nil` is allowed?

Reply:
That's for inputs that lack the `type` attribute, since  they default to `text`.

But looking now I think it will be better to make the input type search default "text" in the `extract_input_values` function.

What do you think?

Reply:
Yeah, makes more sense to me :+1: